charts/argo-cd/values.yaml

```diff
@@ -2028,15 +2029,17 @@ server:
 
     # -- Additional ingress rules
     # @default -- `[]` (See [values.yaml])
+    ## Note: Supports use of custom Helm templates
```

Comment:
Can we move this before the @default line so it is present inside the README?
Maybe we can also use the syntax of Bitnami charts
`(evaluated as a template)`?

Reply:
@mkilchhofer We have this note in other places such as `extraContainers`. Do you want to change that as well?

Reply:
Ah, I forgot that 😄 
Lets continue then 👍